reliquery/storage.py

```diff
@@ -312,36 +291,14 @@ def put_metadata(self, path: Storage, metadata: Dict):
         )
 
     def get_metadata(self, path: StoragePath, root_key: str) -> Dict:
-        dirs = [
-            "arrays",
-            "html",
-            "text",
-            "images",
-            "json",
-            "pandasdf",
-            "files",
-            "notebooks",
-        ]
-
-        data = {
-            root_key: {
-                "arrays": [],
-                "text": [],
-                "html": [],
-                "images": [],
-                "json": [],
-                "pandasdf": [],
-                "files": [],
-                "notebooks": [],
-            }
-        }
+        data = {root_key: {k: [] for k in DATA_TYPES}}
 
         def dict_from_path(path: StoragePath, dirname: str) -> None:
             dirpath = path.copy()
             dirpath.append(dirname)
             dir_keys = self.list_keys(dirpath.copy())
 
-            if len(dir_keys) > 0:
+            if dir_keys:
```

Comment:
do we need the if? i believe it would be ok if list_keys() returns an empty list right?